### PR TITLE
feat(TBD-12623): universal yarn launcher for spark 2.4

### PR DESCRIPTION
**Pull request by AlixMetivier**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

**Review by cbadillo1603 — DISMISSED** [review]
reviewed

File: main/features/org.talend.hadoop.distribution.feature/feature.xml [hunk 1]
```diff
@@ -51,5 +51,6 @@
   <plugin id="org.talend.hadoop.distribution.piv200" download-size="0" install-size="0" version="0.0.0" unpack="false"/>
   <plugin id="org.talend.hadoop.distribution.qubole" download-size="0" install-size="0" version="0.0.0" unpack="false"/>
   <plugin id="org.talend.hadoop.distribution.spl30x" download-size="0" install-size="0" version="0.0.0" unpack="false"/>
+  <plugin id="org.talend.spark.distribution.spark247" download-size="0" install-size="0" version="0.0.0" unpack="false"/>
```
Comment (lbourgeois) [inline main/features/org.talend.hadoop.distribution.feature/feature.xml:54]: Shouldn't it be spark24x (similar to spark31x) to be able to upgrade to spark 2.4.8 when available ?

Comment (AlixMetivier) [reply main/features/org.talend.hadoop.distribution.feature/feature.xml:54]: sure can do, but in the end its just the name of the package, its shown as spark 2.4.x in the UI

Comment (lbourgeois) [reply main/features/org.talend.hadoop.distribution.feature/feature.xml:54]: There are 2 converns here:
- having plugins/packages reflecting their content : as this same plugin should be used to upgrade to 2.4.8 if there is any then plugin/classes should be 24x
- UI display : all  Spark versions are not aligned. Original idea starting from spl30x was to have 3.0.x as key but real targeted version displayed (3.0.1 for the moment - to be replaced by 3.0.2 when upgrade is done) but maybe it does not fit current needs. Maybe the best option here is to display MAJOR.MINOR only.

**Pull request closed without merging**